cluster-autoscaler/cloudprovider/aws/aws_cloud_provider.go

```diff
@@ -227,6 +228,11 @@ func (asg *Asg) Nodes() ([]string, error) {
 	return asg.awsManager.GetAsgNodes(asg)
 }
 
+// TemplateNodeInfo returns a node template for this node group.
```

Comment:
IMHO explaining (why its name is prefixed with `Template` / why it is meant to be a `Template`) would help other developers like me to understand the possible purpose of utilizing this func.